content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md

```diff
@@ -0,0 +1,339 @@
+
+You will learn about the core concepts used in this work, and this will just be sufficient to enable you to build something similar or even more complex.
+
+### Prerequisites
+To better understand and build along with this tutorial you should have:
+
+- The Chrome browser installed.
+- An offline or online code editor (VS Code in my case).
+- Some knowledge of Javascript and its syntax.
+- Your desktops notification turned on - this is where our motivational quotes will be displayed.
+    
+### Table of Contents
+  - [Getting started](#Getting-started)
+  - [Adding icons](#Adding-icons)
+  - [Functionality](#Functionality)
+  - [Creating the background script](#Creating-the-background-script)
+  - [background.js](#background.js)
+  - [Adding Permissions](#Adding-permissions)
+  - [Fetching random quotes from the API](#Fetching-random-quotes-from-the-API)
+  - [Calling the quotes at intervals](#Calling-the-quotes-at-intervals)
+  - [Creating notifications](#Creating-notifications)
+  - [Conclusion](#Conclusion)
+  - [References](#Reference)
+  
+
+### Getting started
+To get started we will need to create a directory to hold the extension's files on our local machine. Using windows command prompt we can type in the following: 
+
+```bash
+mkdir Random-Quote-Extension
+cd Random-Quote-Extension
+code .
+```
+
+The first line creates a new folder in our machine, the second line helps navigate into the folder we just created, and the last line opens up the just created folder in Visual studio code.
+
+Next, we will create a `manifest.json` file and add the following codes to make it look like so:
+
+```json
+{
+    "name": "Random Quote Extension for Chrome",
+    "description": "A Chrome Extension that shows random quotes as notification",
+    "version": "1.0.0",
+    "manifest_version": 3
+}
+```
+
+The `manifest.json` file contains important information about the extension.
+
+- The `name` field contains information about the title of the extension we are building - Random Quote Extension.
+- The `Description` field as the name implies gives a brief description of the functionality of the extension.
+- The `manifest_version` field tells us the current version we are building with. Ensure to use 3.0 as the previous version (2.0) is deprecated
+- Also, it is nice to start with a `version` that is small, you may want to update your extension as time passes. I go with 1.0 for now.
+
+Now that we have our manifest file setup, let us add the directory as an extension in developer mode in our Chrome browser.
+
+Navigate to the Extensions management page by clicking on the extensions menu button at the top right of the browser, and selecting manage extensions at the bottom of the menu.
+
+You should see a page like this:
+![developer](/engineering-education/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/images/developer-mode.PNG)
+
```

Comment:
Uppercase letters are not supported in the article URL. ie. `How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript`. Please replace them with lowercase letters.

Fix all your images.